src/nvim/CMakeLists.txt

```diff
@@ -791,6 +792,22 @@ add_custom_target(
   DEPENDS ${LINT_PRG} ${LINT_NVIM_SOURCES} ${LINT_SUPPRESS_FILE}
 )
 
+add_custom_target(
+  clint-non-style
+  COMMAND
+    ${LINT_PRG} --filter=-whitespace,-readability ${LINT_NVIM_REL_SOURCES} 
+  WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
+  DEPENDS ${LINT_PRG} ${LINT_NVIM_SOURCES} ${LINT_SUPPRESS_FILE}
+)
+
+add_custom_target(
+  clang-tidy-full
```

Comment:
why is this needed? please don't add things just because they're marginally nice-to-have. It increases the surface area that developers have to consume.